src/forms/BookingTimeForm/FieldDateAndTimeInput.js

```diff
@@ -139,7 +139,7 @@ const getAllTimeValues = (
   const startTime = selectedStartTime
     ? selectedStartTime
     : startTimes.length > 0 && startTimes[0] && startTimes[0].timestamp
-    ? startTimes[0].timestamp
+    ? startTimes[0].timestamp.toString()
```

Comment:
This is pretty hard to understand - which is to say that this is an example of one of the hardest problems in programming: naming variables. 

What is the expected type of this "startTime" variable?
- Is it time (aka `Date`)?
- Is it a real timestamp as ternary suggest => i.e. should it be a `Number`
- Or is it a `String` representation of timestamp.

Another dimension is use-case: if I understood correctly, this is the value of the `<option>` that should be selected from start-time `<select>`. 
=> therefore, the value needs to be string.
(It's pretty annoying that there are so many conversions needed here: string -> int -> date)

At least, we should add code-comments, since this is clearly error prone.

> Side note: this whole setup (related to availability management) could benefit of some refactoring, but I just don't have any vision how to do it. :/
This is pretty much the consequence of having deadlines. Improvements and code-iteration doesn't happen.